validating_blockstore.go

```diff
@@ -0,0 +1,31 @@
+package blockstore
+
+import (
+	blocks "github.com/ipfs/go-block-format"
+	cid "github.com/ipfs/go-cid"
+)
+
+// TODO: Validate put?
+
+// ValidatingBlockstore validates blocks on get, if enabled.
+type ValidatingBlockstore struct {
+	Blockstore
+	ValidateGet bool
```

Comment:
The use of the wrapper already serves as this bool. remove it and just decide if you want to wrap an underlying blockstore with this wrapper.

Reply:
I put that there so we could _also_ introduce validations for put. But you're right, that's probably not necessary.